packages/frontend/src/pages/campaigns/index.tsx

```diff
@@ -65,13 +49,10 @@ export const Campaigns = () => {
     useEffect(() => {
         if (loading || kpiTokens.length > 0) return;
         setKpiTokens(Object.values(response));
-        setKpiTokensReady(true);
```

Comment:
I think we should keep `kpiTokensReady`, otherwise the `Empty` component would be rendered for a second, between the time the KPI tokens are fetched (`loading: false`) and the `setKpiTokens` actually completes.